torch/csrc/autograd/python_variable.cpp

```diff
@@ -1436,6 +1446,11 @@ static struct PyGetSetDef THPVariable_properties[] = {
     {"is_cpu", (getter)THPVariable_is_cpu, nullptr, nullptr, nullptr},
     {"is_xpu", (getter)THPVariable_is_xpu, nullptr, nullptr, nullptr},
     {"is_ipu", (getter)THPVariable_is_ipu, nullptr, nullptr, nullptr},
+    {"is_privateuse1",
+     (getter)THPVariable_is_privateuse1,
```

Comment:
@heidongxianhua I think we can probably avoid exposing this as a new python API (and also avoid adding the related extra plumbing, like `TensorImpl::is_privateuse1`).

Is there something that you need this API for? I don't see it used anywhere in the PR. I also don't think this is really something that user code should call - if you really need a way to check if your tensor uses the custom backend, you can check its device.

Reply:
yeah, as for a new device, we may want to assert its device type with tensor.is_foo, and in Python, just alias the func, `torch.Tensor.is_foo = torch.Tensor.is_foo.is_privateuse1`, it may be faster than assert with device, like b.device.type == 'foo'.

Reply:
Thanks! I think the perf difference between a device comparison and `.is_privateuse1` will be negligible compared to python overhead. And if a user is that sensitive to perf, they can run the code in C++ instead.